Exercise/reportexercise.js

```diff
@@ -11,7 +11,7 @@ if( !isVar("VARIABLE_EXERCISE_TIMES"))
 		setDate("VARIABLE_JUST_EXERCISED", tempdate);
 		setVar("VARIABLE_EXERCISE_TIMES", 1);
 	}
-    tempdate2 = getDate("VARIABLE_JUST_EXERCISED");
+    tempdate2 = getDate("LastExercise");

```

Comment:
Renamed variable but didn't change it two lines above.  Btw there is a difference between Strings and global vars:
VARIABLE_SUB_PARTNER_NAME is not the name of the variable. It is the name of the global variable in java script holding the name of the variable (a string)

Reply:
good catch... fixed up 2 instances of diverged variable names